packages/patternfly-4/content/design-guidelines/usage-and-behavior/charts/design.md

```diff
@@ -62,20 +64,58 @@ Stacked bar charts describe totals while allowing a degree of internal breakdown
 <!--#### React component
 [Stacked bar chart](/documentation/react/components/stackchart)-->
 
-### Donut chart
+### Donut charts
 Donut charts represent relative amounts that must add up to 100%.
 
 #### Usage
-The most common use cases for donut charts are to show progress completion as a percentage, show utilization for an object as a percentage and show the relationship of a set of values to a whole. Use donut charts to compare an individual section to the whole. As said previously, the entire donut should add up to 100% and the fill represents a percentage of that. There should be no more than six categories in total, similar to pie charts We recommend that you use the [Colors for charts](/design-guidelines/styles/chart-colors) to represent your data when thresholds are not present. Consider using a [Bar chart](#bar-chart) if you would like to compare one category to another.
+These are the most common use cases for donut charts:
+  - Showing the relationship of a set of values to a whole
+  - Showing progress completion as a percentage
```

Comment:
I question this a bit. While you could use a donut chart for this purpose, it's not what I typically see. If you were trying to display progress of a task, isn't a Progress Bar more appropriate?

Reply:
Should I instead use the word utilization? So:

These are the most common use cases for donut charts:
- Showing the relationship of a set of values to a whole
- Showing utilization as a percentage (with or without thresholds)

Reply:
Yes, I think that's better